### PR TITLE
RT133623 Disable XS build check in pureperl mode

### PR DESCRIPTION
**Pull request by nfg**
RT: https://rt.cpan.org/Public/Bug/Display.html?id=133623

I'm not sure how to build the module from the repo, but this fix works
when applied to the latest tarball.

Let me know if there's anything I should do.

Cheers!

### CODE REVIEW

**Comment by nfg** [comment]
If you get a chance to look at this before the end of the month, I'd be happy if you added the "hacktoberfest-accepted" label ;)

File: inc/Config/AutoConf/ParamsUtil.pm [hunk 1]
```diff
@@ -22,8 +22,11 @@ sub check_paramsutil_prerequisites
 {
     my $self = shift->_get_instance();
 
-    $self->{config}->{cc}                    or $self->check_prog_cc();
-    $self->check_produce_loadable_xs_build() or die "Can't produce loadable XS module";
+    if (!$self->check_pureperl_required())
+    {
+        $self->{config}->{cc}                    or $self->check_prog_cc();
+        $self->check_produce_loadable_xs_build() or die "Can't produce loadable XS module";
```
Comment (rehsack) [inline inc/Config/AutoConf/ParamsUtil.pm:28]: I'd prefer to fallback to pureperl build here - it's a copy'n'paste error.
I can either modify your pull request or you force push an updated patch.

Comment (nfg) [reply inc/Config/AutoConf/ParamsUtil.pm:28]: I can modify the pull request, but I'm not sure how to fallback to pureperl. Is it just a matter of not dying if `$self->check_produce_loadable_xs_build()` fails?

Comment (rehsack) [reply inc/Config/AutoConf/ParamsUtil.pm:28]: What do you think about https://github.com/perl5-utils/Params-Util/commits/nfg-rt133623 ?

Comment (nfg) [reply inc/Config/AutoConf/ParamsUtil.pm]: Looks good to me! I cherry-picked the new commit and pushed it up.

Comment (nfg) [reply inc/Config/AutoConf/ParamsUtil.pm]: Or you can just merge that nfg-rt133623 branch you made. Whatever works for you.

Comment (rehsack) [reply inc/Config/AutoConf/ParamsUtil.pm]: I check it tomorrow morning and do a release then ;)

Comment (nfg) [reply inc/Config/AutoConf/ParamsUtil.pm]: Cool. Thanks so much!

**Comment by rehsack** [comment]
I merged my `nfg-rt133623` branch and prepare for new release right now before I forget it again ;)

**Pull request closed without merging**